tests/sessions_tests/tests.py

```diff
@@ -886,11 +886,6 @@ def test_cycle(self):
         """
         pass
 
-    @unittest.expectedFailure
-    def test_actual_expiry(self):
-        # The cookie backend doesn't handle non-default expiry dates, see #19201
-        super().test_actual_expiry()
```

Comment:
We should remove `@unittest.expectedFailure` decorator instead of removing the test:
```python
    def test_actual_expiry(self):
        super().test_actual_expiry()
```

Reply:
This doesn't remove the test, it actually enables the test by removing the override of the function from SessionTestsMixin.

Reply:
Right, thanks :+1: